packages/datadog-plugin-pg/src/index.js

```diff
@@ -32,9 +32,15 @@ function createWrapQuery (tracer, config) {
       }
 
       const originalCallback = pgQuery.callback
-      const statement = pgQuery.text
+      const statement = (pgQuery.cursor && pgQuery.cursor.text) || pgQuery.text
       const params = this.connectionParameters
 
+      if (pgQuery.readable) {
+        // pgQuery is a readable stream (using a cursor)
+        pgQuery.on('close', function() { pgQuery.callback() })
```

Comment:
Can this cause issues since the callback is set internally by `pg`? If the idea is to finish the span, maybe that could be done explicitly instead?